openmmtools/multistate/sams.py

```diff
@@ -363,8 +363,10 @@ def _pre_write_create(self, thermodynamic_states: list, sampler_states: list, st
         # Update log weights
         self._update_log_weights()
 
-    def _restore_sampler_from_reporter(self, reporter):
-        super()._restore_sampler_from_reporter(reporter)
+    def _restore_sampler_from_reporter(self, reporter, energy_context_cache=None, propagation_context_cache=None):
```

Comment:
Should we just use `**kwargs` so any kwargs can be passed to the base class?

Reply:
Using arbitrary keyword arguments (kwargs) now. Do we want to do the same for non-keyword arguments(`reporter`) here? Seems like it might make it less readable :thinking: